x/bridge/observer/osmosis/client.go

```diff
@@ -145,6 +149,28 @@ func (c *Client) Account(ctx context.Context, addr sdk.AccAddress) (authtypes.Ba
 	return ba, nil
 }
 
+func (c *Client) ConfirmationsRequired(
+	ctx context.Context,
+	assetId bridgetypes.AssetID,
+) (uint64, error) {
+	req := bridgetypes.QueryParamsRequest{}
+	params, err := c.bridgeClient.Params(ctx, &req)
+	if err != nil {
+		return 0, errorsmod.Wrapf(ErrQuery, "bridge/params: %s", err.Error())
+	}
+	idx := slices.IndexFunc(params.GetParams().Assets, func(a bridgetypes.Asset) bool {
+		return a.Id == assetId
+	})
+	if idx == -1 {
```

Comment:
It's better to wrap all magic numbers into constants. There is even a linter for that, which is unfortunately disabled in osmosis
```suggestion
	const idxNotFound = -1
	if idx == idxNotFound {
```